### PR TITLE
Feat/staging deltas

### PR DESCRIPTION
**Pull request by farski**

### CODE REVIEW

File: cd/cd.yaml [hunk 1]
```diff
@@ -528,14 +528,14 @@ Resources:
...
-            - Name: StagingDeploy
+            - Name: CreateStagingChangeSet
               ActionTypeId:
                 Category: Deploy
                 Owner: AWS
                 Provider: CloudFormation
                 Version: 1
               Configuration:
-                ActionMode: REPLACE_ON_FAILURE
+                ActionMode: CHANGE_SET_REPLACE
```
Comment (cavis) [inline cd/cd.yaml]: Is this an unrelated change?

Comment (cavis) [reply cd/cd.yaml]: Or is this just to separate creating/executing the changeset, so you can notify in between?

Comment (farski) [reply cd/cd.yaml]: oh hm

Comment (farski) [reply cd/cd.yaml]: that probably should change back. replace is just a more aggressive method, so if staging were to get stuck in a bad way another deploy could just replace it, rather than trying to update which always needs to happen from a stable state
